Islandora-CLAW/islandora_demo
# pdf.js viewer configs

# What does this Pull Request do?
This PR adds pdf viewer to islandora_demo.

# What's new?
The steps for creating this viewer are described [here](https://github.com/Natkeeran/Notes/blob/master/configure-new-viewer-in-claw.md).

# How should this be tested?
* Vagrant up
* Install drupal module [pdf](https://www.drupal.org/project/pdf).  Make sure to have the pdf.js library.  (This will be a separate PR).
* Get this PR 
* Run the migration `/admin/structure/migrate/manage/islandora/migrations/islandora_demo_tags` (adds one more display type term)
* Import the changes `/admin/config/development/features/diff/islandora_demo`
* Create a Repository object of binary type, Choose PDFjs for display hint
* Add a media (file), Set Media Use as Original (can be other types as well)
* Ensure that the pdf is displayed using the pdf.js viewer when viewed from the node


# Additional Notes:
* https://github.com/Natkeeran/islandora_pdf/issues/1

# Interested parties
@kayakr @MarcusBarnes  @dannylamb 

@Islandora-CLAW/committers
